receptor-python-worker/setup.cfg

```
@@ -1,5 +1,5 @@
 [metadata]
-name = receptor-python-worker
+name = receptor_python_worker
```

Why do you want to change it? It's usually good to keep the project name separate from the importable name.

---

I also noticed it diverges from the mirror setting in `pyproject.toml`.
```suggestion
name = receptor-python-worker
```